src/containers/BrandsForm/BrandsForm.js

```diff
@@ -1,19 +1,83 @@
-import React from "react";
-
+import React, { useState } from "react";
 import { Button, TextField } from "@material-ui/core";
-
 import "./style.css";
 
-const BrandsForm = () => (
-  <section>
-    <form className="brandsForm" noValidate autoComplete="off">
-      <div className="brandsForm--inputs">
-        <TextField id="standard-basic" label="Marca" />
-      </div>
-      ß<Button variant="contained">Cadastrar</Button>
-      <Button variant="contained">Cancelar</Button>
-    </form>
-  </section>
-);
+const BrandsForm = ({ sendBrand }) => {
+  const [brand, setBrand] = useState("");
+  const [errors, setErrors] = useState({
+    validator: { isvalid: true, text: "" }
+  });
+
+  function validate(brand) {
+    let isBrandValid = {
+      isvalid: true,
+      text: "",
+    };
+
+    if (brand.length <= 2) {
+      isBrandValid = {
+        isvalid: false,
+        text: "A marca deve ter pelo menos 3 dígitos",
+      };
+    }
+
+    if (brand.length === 0) {
+      isBrandValid = {
+        isvalid: false,
+        text: "Preencha o campo Marca",
+      };
+    }
+
+    return isBrandValid;
+  }
+
+  function sendForm(brand) {
+    if (brand.length === 0) {
+      return false;
+    }
+    sendBrand(brand);
+  }
+
+  return (
+    <section>
+      <form onSubmit={
+        (event) => {
+          event.preventDefault();
+          setErrors({ validator: validate(brand) });
+          sendForm(brand)
+        }
+      } className="brandsForm" noValidate autoComplete="off">
+        <div className="brandsForm--inputs">
+          <TextField
+            onChange={(event) => {
+              setBrand(event.target.value);
+            }} value={brand}
```

Comment:
Tem um warning no console sobre a dupla declaração desse attribute. Pode remover esse!
```suggestion
            }}
```

Reply:
dei merge nao vi esse comentario '-' mals ae

Reply:
vou fazer agora e ja envio numa branch nova